myClasses/UVSim_Class.py

```diff
@@ -14,11 +15,15 @@ def __init__(self):
 
     def load(self, filename):
         '''Load Instructions Into Memory'''
-        if not os.path.exists(filename):
-            print(f"File '{filename}' does not exist.")
+        current_dir = os.getcwd()
+        current_dir = current_dir + "\Test Files\""
+        relative_path += filename
```

Comment:
This is breaking the program when its ran from main.py.
cannot access local variable 'relative_path' where it is not associated with a value
  File "/Users/danielwilsher/Documents/cs2450/myClasses/UVSim_Class.py", line 20, in load
    relative_path += filename
    ^^^^^^^^^^^^^
  File "/Users/danielwilsher/Documents/cs2450/myClasses/UVSim_Class.py", line 46, in execute
    self.load(filename)
  File "/Users/danielwilsher/Documents/cs2450/main.py", line 7, in main
    uvsim.execute()
  File "/Users/danielwilsher/Documents/cs2450/main.py", line 10, in <module>
    main()
UnboundLocalError: cannot access local variable 'relative_path' where it is not associated with a value

Reply:
I believe the variable needs to be declared first